sdk/examples/client/stronghold.rs

```diff
@@ -15,7 +15,7 @@ use iota_sdk::client::{
 #[tokio::main]
 async fn main() -> Result<()> {
     let stronghold_secret_manager = StrongholdSecretManager::builder()
-        .password("some_hopefully_secure_password")
+        .password("some_hopefully_secure_password".to_owned())
```

Comment:
Maybe impl `From<&str>` for `Password` so the UX isn't degraded?

Reply:
The problem is that this encourages passing in referenced data which will not be zeroized. Tbh the whole thing is highly dependent on the user managing their data properly, but I think forcing owned data here is better.

Reply:
But these are string literals they'll end up in the binary and loaded in memory so not zeroized anyway

Reply:
I thought about that, but considered it too much of a risk just to make the examples and tests more convenient. In actual code the user would never have a `&static str` for a password, but instead get an owned `String` from somewhere. If we allow `&str`, then the user might make the error of providing just a reference to the String, and forget about zeroing the original on his own. He could also just clone the String and also forget about zeroing the original, but I consider that to be acceptable, because we did what we could to protect his/her secret.

Reply:
oh, shoot, browser didn't update the discussion ... exactly what Alex has said is my point as well.